### PR TITLE
Make url field in afterFillSettings of form wizard unnecessary 

### PR DESCRIPTION
**Pull request by radiansz**

### CODE REVIEW

File: src/resources/form/model.js [hunk 1]
```diff
@@ -314,6 +314,9 @@ define(function(require) {
             if (this.get('fillLink')) {
                 data.fill = data.fill || {};
                 data.fill.link = this.get('fillLink').get('id');
+            } else {
+                data.fill = data.fill || {};
+                data.fill.link = "";
```
Comment (zouxuoz) [inline src/resources/form/model.js]: wrong quotes, we are using `'`

File: src/resources/form/model.js [hunk 2]
```diff
@@ -308,12 +308,15 @@ define(function(require) {
...
                 data.welcome = data.welcome || {};
-                data.welcome.draft = "";
+                data.welcome.draft = '';
             }
 
             if (this.get('fillLink')) {
                 data.fill = data.fill || {};
                 data.fill.link = this.get('fillLink').get('id');
+            } else {
+                data.fill = data.fill || {};
```
Comment (rosendi) [inline src/resources/form/model.js]: Same as line 315. Unwrap it.

File: src/blocks/form/formAfterFillSettings/formAfterFillSettings.js [hunk 3]
```diff
@@ -121,7 +121,12 @@ define(function(require) {
                 this.models.form.unset('welcomeDraft');
             }
 
-            this.models.form.get('fillLink').set('url', data.fillUrl);
+            if (data.fillUrl.trim() !== '') {
+                this.models.form.get('fillLink').set('url', data.fillUrl);
+            }
+            else {
```
Comment (rosendi) [inline src/blocks/form/formAfterFillSettings/formAfterFillSettings.js]: Merge lines 126 and 127

File: src/blocks/form/formAfterFillSettings/formAfterFillSettings.js [hunk 4]
```diff
@@ -121,7 +121,12 @@ define(function(require) {
                 this.models.form.unset('welcomeDraft');
             }
 
-            this.models.form.get('fillLink').set('url', data.fillUrl);
+            if (data.fillUrl.trim() !== '') {
```
Comment (rosendi) [inline src/blocks/form/formAfterFillSettings/formAfterFillSettings.js]: Can it be just `this.models.form.get('fillLink').set('url', data.fillUrl);`. Why have we unset `fillLink`?

Comment (radiansz) [inline src/blocks/form/formAfterFillSettings/formAfterFillSettings.js]: fillLink is api entity, it's url can not be setted to ''. There was a mistake with initializing of 'fillLink' in initialize function. It shouldnt be initialized there considering fillLink is unnecessary.

**Comment by rosendi** [comment]
LGTM

**Pull request merged**